packages/@blockprotocol/graph/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@blockprotocol/graph",
-  "version": "0.0.10",
+  "version": "0.0.11",
```

Comment:
What’s the purpose of this bump? We could leverage `changesets` instead.

Reply:
Oh I see we refer to a new version in `packages/block-template/templates/custom-element/package.json`. I guess we don't need `.changeset/convert-graph-to-esm-package.md` then.